tests/06-lb.sh

```diff
@@ -430,11 +431,11 @@ docker exec -i client ping -c 4 $LB_HOST_IP4 || {
 	abort "Error: Unable to reach local IPv4 node via loadbalancer"
 }
 
-sudo cilium endpoint ct dump $SERVER1_ID
+#sudo cilium endpoint ct dump $SERVER1_ID
```

Comment:
All commands which directly access bpf maps have been grouped in the bpf command to make it clear what uses the remote API and what uses direct access APIs